packages/server/api/src/app/ai/config/ai-config.service.ts

```diff
@@ -48,6 +54,26 @@ export const aiConfigService = {
 
     const config = await repo().save(aiConfig);
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let telemetryConfig: any = {
+      id: config.id,
+      userId: params.userId,
+      projectId: params.projectId,
+      model: config.model,
+      provider: config.provider,
+    };
+
+    if (config.provider != AiProviderEnum.AZURE_OPENAI) {
```

Comment:
@bigfluffycookie maybe instead let's exclude sending the model.
And report all providers.